File: src/main/java/org/example/gogoma/controller/request/MarathonEndInitDataRequest.java
package org.example.gogoma.controller.request;

import lombok.*;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class MarathonEndInitDataRequest {
    private int currentPace;
    private int runningTime;
    private int totalMemberCount;
    private int myRank;
}

File: app/src/main/java/com/example/gogoma/data/model/MarathonEndInitDataRequest.kt
package com.example.gogoma.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class MarathonEndInitDataRequest(
    val currentPace: Int,
    val runningTime: Int,
    val totalMemberCount: Int,
    val myRank: Int
) : Parcelable